core/src/testUtils/groovy/nu/yona/server/test/AnalysisService.groovy

```diff
@@ -40,8 +39,38 @@ class AnalysisService extends Service
 					"url":"$url"
 				}""")
 	}
+
 	def postToAnalysisEngine(String vpnLoginId, jsonString)
 	{
 		yonaServer.postJson(USER_ANONYMIZED_PATH + vpnLoginId + "/networkActivity/", jsonString)
 	}
+	void setEnableStatistics(def enable)
```

Comment:
This part is duplicate code with `AppService`. Can't we merge this to `YonaServer`?

Reply:
Makes sense. Moved it.